Protocol.md

```diff
@@ -1,7 +1,7 @@
 ## Status
 
 This protocol is currently a draft for the final specifications. 
-Current version of the protocol is __0.1__ (Major Version: 0, Minor Version: 1).
+Current version of the protocol is __1.0__ (Major Version: 1, Minor Version: 0).
```

Comment:
I'm okay with this remaining something like 0.2 or 0.10 for now if that makes life easier. Then it will be a single PR to change only this once we have all validated it in Jan/Feb.